topics/metagenomics/tutorials/cami2-assembly/tutorial.md

```diff
@@ -0,0 +1,1085 @@
+
+| CAMI benchmark dataset                                               | DOI                                              |
+|--------------------------------------------------------------------- |------------------------------------------------- |
+| CAMI I: low, medium, high complexity, and ‘toy’ datasets             | 10.5524/100344                                   |
+| CAMI II: human microbiome project and mouse gut toy datasets         | 10.4126/FRL01-006425518; 10.4126/FRL01-006421672 |
+| CAMI II: marine, strain madness, rhizosphere, and pathogen detection | 10.4126/FRL01-006425521                          |
+
+
+> ### {% icon hands_on %} Hands-on: Get dataset with DOI
+> 
+> 1. Use the table above to find unique identifier (DOI) of dataset you need
+> 2. Use [https://search.datacite.org/](https://search.datacite.org/) to search for data via DOI, and click the link that was found for the DOI. 
+> 3. On the page of dataset there is the link provided to download dataset -  for marine dataset all samples available via [https://frl.publisso.de/data/frl:6425521/marine/](https://frl.publisso.de/data/frl:6425521/marine/)
+> 4. Download datasets on your computer
+{: .hands_on}
+
+
+> ### {% icon details %} Download datasets with CAMI Client
+>
+> Another way to download datasets is to use CAMI Client. [Here](https://www.microbiome-cosi.org/cami/resources/cami-client) you can find instructions on how to use it.
+>
+> > ### {% icon hands_on %} Hands-on: Get dataset with DOI
+> > 
+> > 1. Install Java 8 on your computer.
+> > 2. Create an account on[ CAMI portal](https://data.cami-challenge.org/)
+> > 3. Download the [camiClient.jar](https://data.cami-challenge.org/camiClient.jar)
+> > 4. Get text-file with links for dataset from [CAMI portal](https://data.cami-challenge.org/). You can download this file in the section “2nd CAMI Challenge Marine Dataset”
+> > 5. You can download data with CAMI client using the following command line in the terminal:
+> > 
+> >     ```
+> > java -jar camiClient.jar -d <linkfileLocation> <targetDirectory> -p PATTERN
+> > ```
+> {: .hands_on}
+> 
+{: .details}
+
+
+#### Upload data into Galaxy
+
+There are different ways to upload data into the Galaxy. In case of a not very large dataset it’s more convenient to upload data directly from your computer to Galaxy. In this tutorial we use a large dataset, thus, we use FTP server for this purpose.
+
+> ### {% icon hands_on %} Hands-on: Upload data into Galaxy with FTP
+>
+> 1. Create new history
+>
+>    {% snippet faqs/galaxy/histories_create_new.md %}
+>
+> 2. Upload datasets into FTP Galaxy server
+>
+>    {% snippet faqs/galaxy/datasets_upload_ftp.md %}
+>
+> 3. Press “upload data” -> “choose remote files” -> “FTP directory” and upload your data to your history from FTP server
```

Comment:
The rule builder would remove these steps which would make it *much* easier for folks to follow this tutorial, FTP is a big complication for users.

Reply:
Do you mean, it would be better if I remove these steps from the tutorial?

Reply:
Yes, remove, and replace with the Rule Builder to upload by URLs. E.g. https://training.galaxyproject.org/training-material/topics/transcriptomics/tutorials/rna-seq-reads-to-counts/tutorial.html#import-data-from-urls they provide a text table that users can copy-paste into Galaxy, and below is a set of instructions for using the rule based uploader to turn those into datasets in a history.

Reply:
Thank you, I just learned this way, and it's way more convenient than ftp. However, in case of CAMI2 datasets links, every time you should download the file with temporary links for datasets. They are different every time and have an expire date. So, maybe it's better to ask a Galaxy user to create this text table with links by their own and then to use it in Rule Builder?

Reply:
Oh wow that's difficult! It makes sense for the challenge though.

1. Yes, I think that's a good idea, show them an example of the table and then tell them to construct their own or so.
1. are these datasets restricted, is there something like a "you may not share these publicly" applied? If not we could put a static set of them on zenodo so that people could follow the tutorial reproducibly?

Reply:
I changed the tutorial according to the 1st option.